File: CMS/src/app/shared/DanhMucGiong.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { DanhMucGiong } from 'src/app/shared/DanhMucGiong.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class DanhMucGiongService extends BaseService{
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "DanhMucGiong";
    }
}

File: Service/Implement/DanhMucGiongService.cs
namespace Service.Implement
{
    public class DanhMucGiongService : BaseService<DanhMucGiong, IDanhMucGiongRepository>
    , IDanhMucGiongService
    {
    private readonly IDanhMucGiongRepository _DanhMucGiongRepository;
    public DanhMucGiongService(IDanhMucGiongRepository DanhMucGiongRepository) : base(DanhMucGiongRepository)
    {
    _DanhMucGiongRepository = DanhMucGiongRepository;
    }
    }
    }

